pg_exclude.yml

```diff
@@ -0,0 +1,5 @@
+exclude:
+  nightly:
+    ol/7: [15]
+  release:
+    all: [15]
```

Comment:
How about moving this file and merging it with some of the current files that we already have? See https://github.com/citusdata/tools/pull/238#discussion_r941533392

Reply:
Actually we will not need this file quite common so I prefer not adding this option into postgres_matrix file. Additionally, postgres matrix includes all versions which is some sort of a citus postgres version support matrix file. This file will be very long and adding additional things will make the file more difficult to read.
However, it is not a bad idea. If it is needed, I need to open a PR about that and merge them in the future since there are more prioritized tasks waiting my attention in my work queue.

Reply:
I think that it is not too much of a hassle to concatenate the two files. I do not think that you need to update the structure.

How about the following suggestion for `postgres-matrix.yml`:

```yml
name: Postgres Version Matrix
project_name: citus

version_matrix:
  - 8.0:
      postgres_versions: [ 10, 11 ]
  - 9.0:
      postgres_versions: [ 11, 12 ]
  - 9.5:
      postgres_versions: [ 11, 12, 13 ]
  - 10.1:
      postgres_versions: [ 12, 13 ]
  - 10.2:
      postgres_versions: [ 12, 13, 14 ]
  - 11.0:
      postgres_versions: [ 13, 14 ]
  - 11.1:
      postgres_versions: [ 13, 14, 15 ]

exclude:
  nightly:
    ol/7: [ 15 ]
  release:
    all: [ 15 ]
```

Reply:
Yes it can be like that but I wanted to keep postgres-matrix.yml for only versions. 
I opened an issue about that. I could start this issue after my prioritized tasks
https://github.com/citusdata/packaging/issues/958